addons/medical/functions/fnc_handleUnitVitals.sqf

```diff
@@ -13,12 +13,15 @@
 
 #include "script_component.hpp"
 
-private ["_unit", "_heartRate","_bloodPressure","_bloodVolume","_painStatus", "_lastTimeValuesSynced", "_syncValues", "_airwayStatus", "_blood", "_bloodPressureH", "_bloodPressureL", "_interval"];
+private ["_unit", "_heartRate","_bloodPressure","_bloodVolume","_painStatus", "_lastTimeValuesSynced", "_syncValues", "_airwayStatus", "_blood", "_bloodPressureH", "_bloodPressureL", "_interval", "_lastMomentVitalsHandled"];
 _unit = _this select 0;
 
-_interval = time - (_unit getVariable [QGVAR(lastMomentVitalsHandled), 0]);
```

Comment:
Can't you do: `_interval = time - (_unit getVariable [QGVAR(lastMomentVitalsHandled), time - 1]);`?